### DISCUSSION TITLE
Is there a way to specify `architecture: "x86"` when using this action?

### DISCUSSION THREAD

**Discussion by hirehamir** [LizardByte/community#4] [Actions] [answered]
I'm looking into fixing https://github.com/dictation-toolbox/Caster/actions/runs/7824447503/job/21347010305#step:3:9 but their CI uses `architecture: 'x86'`.

**Comment by ReenigneArcher** [comment]
No, it's not supported right now. A PR for this would be welcomed.

**Reply by hirehamir** [reply]
Hmm... It looks like adding an `architecture` option wouldn't fix the https://github.com/dictation-toolbox/Caster/actions/runs/7824447503/job/21347010305#step:3:9 error anyway.

```
Run hamirmahal/setup-python-action@feat/allow-architecture-specification
Run actions/setup-python@v[5](https://github.com/hamirmahal/Caster/actions/runs/8458595418/job/23173186558#step:3:6)
Installed versions
  Warning: The support for python 2.7 was removed on June 19, 2023. Related issue: https://github.com/actions/setup-python/issues/672
  Version 2.7.x was not found in the local cache
  Error: The version '2.7.x' with architecture 'x86' was not found for Windows 2022.
  The list of all available versions can be found here: https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json
```

**Reply by ReenigneArcher** [reply, marked as answer]
Try the latest release. It accepts the `architecture` input on Windows for Python 2.7, and for Python 3+ it will pass the input to the original action.

**Reply by hirehamir** [reply]
Thank you!
